### ISSUE TITLE
Failed to open theme extdata with extdataID=0x000002ce, retval: 0xc8804464

### ISSUE THREAD

**Issue by gyves76** [yellows8/3ds_homemenuhax#261]
[menuhax_manager.zip](https://github.com/yellows8/3ds_homemenuhax/files/801572/menuhax_manager.zip)

Error occured during the app executing:
Failed to open theme extdata with extdataID=0x000002ce, retval: 0xc8804464.

**Comment by Committee** [comment]
did you open/change your theme at least once before installing menuhax?

**Comment by gyves76** [comment]
no, never. I need to do it before? I have followed this tutorial, http://www.logic-sunrise.com/dossiers-et-tutoriaux-833616-3ds-downgradez-votre-3ds-103-en-92.html, the theme step is done after the menuhax_manager step

**Comment by Committee** [comment]
yes, and use https://3ds.guide/ instead of that one to downgrade your 3ds.
